data/defaults.yaml

```diff
@@ -320,3 +320,12 @@ prometheus::grok_exporter::package_ensure: 'latest'
 prometheus::grok_exporter::package_name: 'grok_exporter'
 prometheus::grok_exporter::user: 'grok-exporter'
 prometheus::grok_exporter::version: '1.0.0.RC4'
+prometheus::nginx_prometheus_exporter::scrape_uri: 'http://localhost:8080/stub_status'
```

Comment:
can you please place the static data in the nginx_prometheus_exporter.pp instead of the defaults.yaml? that enables puppet-strings to display it in the REFERENCE.md. We want to move the other data away as well, but lacking time at the moment.

Reply:
sure